apps/nouns-camp/src/nouns-subgraph.js

```diff
@@ -316,6 +319,72 @@ query {
   }
 }`;
 
+const createProposalCandidatesBySignatureAccountQuery = (
```

Comment:
Should this be called `createProposalCandidateSignaturesByAccountQuery`?

Reply:
good point! long names 😞